sdk-python/livescoreapi.py

```diff
@@ -10,11 +10,50 @@
 
 
 class livescores:
+    """
+    A class used to get livescores. 
+    ...
+
+    Attributes
+    ----------
+    key : str
+        an alphanumeric string used to access data from the LiveScore API
+    secret : str
+        an alphanumeric string used to access data from the LiveScore API
+    
+    Methods
+    -------
+    _get_html_file(link)
+    
+    get_all()
+    
+    """
+    def __init__(self,key,secret):
+        self.key = key 
+        self.secret = secret 
+    
+    def _get_html_file(self,link):
+        """returns the source code given a link"""
+    
+        try:
+            html_file = urllib.request.urlopen(link)
+            return html_file
+        except Exception as e:
+            print(str(e))
+            
+    def get_all(self):
+        """returns all the livescores"""
+        
+        link = "http://livescore-api.com/api-client/scores/live.json?key={}&secret={}".format(self.key,self.secret)
```

Comment:
I would put this part of the url in a member variable as well: `http://livescore-api.com/api-client/` imagine we add option to sandbox/test environment (`test.livescore-api.com`) then the SDK will not work with it